docs/project/copyright.md

```diff
@@ -14,7 +14,7 @@ We use the following approach for code contributions:
 - Contributions (like ports) can be based on existing code or algorithms that are licensed with a permissive [OSI-approved licenses](https://opensource.org/licenses) (for example, [MIT](https://opensource.org/licenses/MIT), [Apache 2](https://opensource.org/licenses/Apache-2.0), and [BSD](https://opensource.org/licenses/BSD-3-Clause)) and should be ackowledged in the [repo THIRD-PARTY-NOTICE](/THIRD-PARTY-NOTICES.TXT) file.
 - Non-product contributions (like tests) can be based on existing code or algorithms that are licensed with any OSI-approved license (including [GPL](https://opensource.org/licenses/GPL-2.0)) and should be acknowleged in a local THIRD-PARTY-NOTICE file (for example, [SciMark/THIRD-PARTY-NOTICES](/src/tests/JIT/Performance/CodeQuality/SciMark/THIRD-PARTY-NOTICES)).
 - We will typically reject the use of product code licensed with [Creative Commons](https://creativecommons.org/) or [public domain licenses](https://en.wikipedia.org/wiki/Public-domain-equivalent_license) given their lack of universal acceptance.
-- The [.NET Foundation Contribution License Agreement (CLA)](https://cla.dotnetfoundation.org) is used to license contributions from contributors.
+- The [.NET Foundation Contribution License Agreement (CLA)](https://github.com/dotnet-foundation/foundation/blob/master/guidance/net-foundation-contribution-license-agreement.pdf) is used to license contributions from contributors.
```

Comment:
Is this the correct link? The link goes to archived repo, and the PDF has a "SAMPLE FOR INFORMATION ONLY" water mark.

Reply:
Oops; thanks. I had grabbed the link from opensource.microsoft.com/cla without looking too closely.

I can't find a definitive CLA from the DNF site either. I'll reach out to some DNF folks for guidance.

Reply:
There isn't a rich-text format version of the CLA published online. It's only available as part of the YAML config file here:
https://github.com/dotnet/.github/blob/main/policies/cla.yml

Instead of linking out to that, I've just described that the CLA will be presented once a PR is created.